pom.xml

```diff
@@ -555,6 +556,11 @@
         <artifactId>alpn-boot</artifactId>
         <version>${jetty.alpn.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.twitter</groupId>
+        <artifactId>hpack</artifactId>
+        <version>0.9.1</version>
```

Comment:
My bad, I did not update the pom.xml to [commit hash](https://github.com/netty/netty/commit/14039623137f2bfdaec9a7228e72fe94c663ded5) (which is the commit I have backported from).
Fixed it now to 0.10.0.